docs/modules/ROOT/pages/index.adoc

```diff
@@ -938,6 +938,88 @@ For more information about custom scripts, see xref:scripts.adoc[Custom Scripts
 Transaction-related variables (`transaction_from`, `transaction_to`, `transaction_value`) are not available for Stellar networks.
 ====
 
+==== Message Formatting
+
+Slack, Discord, Telegram, Email and Webhook support Markdown formatting in their message bodies. You can use Markdown syntax to enhance your notifications.
+
+===== Example Email Notification with Markdown
+[source,json]
+----
+{
+  "email_notification": {
+    "name": "Formatted Alert",
+    "trigger_type": "email",
+    "config": {
+      "host": "smtp.example.com",
+      "port": 465,
+      "username": "alerts@example.com",
+      "password": "password",
```

Comment:
Potentially this will require update once env variables support lands.

I guess it makes sense to support env for password field for example